hail/Makefile

```diff
@@ -129,9 +129,9 @@ check-pypi:
 HAIL_TWINE_CREDS_FOLDER ?= /secrets/
 
 .PHONY: pypi-deploy
-pypi-deploy: export TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username)
-pypi-deploy: export TWINE_PASSWORD := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-password)
 pypi-deploy: check-pypi wheel
+	export TWINE_USERNAME := $(shell cat $(HAIL_TWINE_CREDS_FOLDER)/pypi-username) \
```

Comment:
I think these shouldn't have `export`, this is the same as doing 

```
FOO=1 BAR=2 twine upload
```

Not sure the export is valid bash syntax here.

Reply:
Yup, good catch. Fixed.